specs/phase0/fork-choice.md

```diff
@@ -182,12 +182,13 @@ def get_latest_attesting_balance(store: Store, root: Root) -> Gwei:
             and get_ancestor(store, store.latest_messages[i].root, store.blocks[root].slot) == root)
     ))
     proposer_score = Gwei(0)
-    if store.proposer_boost_root != Root() and root == store.proposer_boost_root:
```

Comment:
This conditional implies that `get_latest_attesting_balance` applies the proposer score boost only to the proposer's block, but not its ancestors. This is not the correct method to apply the proposer score boost.

The boost should be applied like usual LMD votes (i.e., `store.latest_messages`) - all ancestors of `store.proposer_boost_root` should get the additional weight.